build.gradle

```diff
@@ -37,15 +37,15 @@ run {
 
 defaultTasks "build"
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = 1.8
```

Comment:
Why is this needed? We should be able to support 1.7 as other Frege projects are still on 1.7.